python/afdko/makeotf.py

```diff
@@ -2583,41 +2628,29 @@ def runMakeOTF(makeOTFParams):
                   "is not T/F" % optionEntry[1])
         elif isinstance(val, list):
             for item in val:
-                params.append('%s "%s"' % (optionEntry[1], item))
+                params.append(optionEntry[1])
+                params.append(item)
         else:
-            params.append('%s "%s"' % (optionEntry[1], val))
+            params.append(optionEntry[1])
+            params.append(val)
 
-    commandString = " ".join(params)
     if makeOTFParams.verbose:
         print("makeotf [Note] Running %s with commands:" %
               os.path.basename('makeotfexe'))
-        print("   cd \"%s\"" % fontDir)
-        print("   %s" % commandString)
+        print(f'   cd "{fontDir}"')
+        print(f"   {params}")
 
-    fdkutils.runShellCmdLogging(commandString)
+    success = fdkutils.run_shell_command(params, suppress_output=True)
```

Comment:
On a closer look, the output cannot be suppressed because it suppresses all the errors and warnings issued by **makeotfexe**. Run this for example:

    makeotf -f makeotf_data/input/cidfont.ps

This is why I wanted to run a few commands first; the test suite doesn't tell the full story.